packages/modules/web_themes/colors/source/src/components/priceChart/GlobalPriceChart.vue

```diff
@@ -143,7 +142,9 @@ const draw = computed(() => {
 	xAxis
 		.selectAll('.tick line')
 		.attr('stroke', 'var(--color-bg)')
-		.attr('stroke-width',(d) => ((d as Date).getHours()%6==0)?'2' :'0.5')
+		.attr('stroke-width', (d) =>
+			(d as Date).getHours() % 6 == 0 ? '2' : '0.5',
```

Comment:
Ist das Komma am Ende von einem Test übrig geblieben? Wenn nicht, was bewirkt es

Reply:
In Javascript ist es zulässig, ein Komma am Ende einer Liste stehen zu lassen. Ich habe den prettier so eingestellt, dass er das immer einfügt. Deswegen wirst Du das überall im Code finden. 

[https://eslint.style/rules/js/comma-dangle](url)